packages/core/upload/admin/src/components/TableList/PreviewCell.tsx

```diff
@@ -86,9 +96,51 @@ export const PreviewCell = ({ type, content }: PreviewCellProps) => {
     );
   }
 
+  if (mime?.includes(AssetType.Audio)) {
+    return (
+      <Flex
+        background="neutral100"
+        borderRadius="100%"
+        color="neutral500"
+        width="3.2rem"
+        height="3.2rem"
+        justifyContent="center"
+      >
+        <VolumeUp width={16} height={16} />
+      </Flex>
+    );
+  }
+
+  let DocIcon;
+
+  switch (fileExtension) {
```

Comment:
Instead of using a switch you can try to create a map like this one
`const DOC_ICON_MAP: Record<string, React.ComponentType<any>> = {
  pdf: FilePdf,
  csv: FileCsv,
  xls: FileXls,
  zip: FileZip,
} as const;`
and handle the document types in this way 
`const fileExtension = getFileExtension(ext);
  const DocIcon = DOC_ICON_MAP[fileExtension] ?? FileIcon;
  return (
    <Flex
      justifyContent="center"
      borderRadius="100%"
      background="neutral100"
      color="neutral500"
      width="3.2rem"
      height="3.2rem"
    >
      <DocIcon width={16} height={16} />
    </Flex>
  );`